### ISSUE TITLE
documentation for the code

### ISSUE THREAD

**Issue by eread-usgs** [USGS-VIZLAB/internal-analytics#160]
We got a request from WIM to provide a readme for this app. and it'd be good to have something in place, very minimal at least. 

 @wdwatkins, would you be willing to take this on?

**Comment by wdwatkins** [comment]
So this would be from a developer standpoint?  If someone wanted to set this up with their own google analytics and AWS accounts?

**Comment by eread-usgs** [comment]
yes i think so, or contributors to it, e.g., if someone wanted a new project added but needed to know which yaml file to add it to, I think it can be pretty minimal, but the readme right now is just blank.
